src/routes/solid-router/reference/data-apis/action.mdx

```diff
@@ -100,3 +101,69 @@ type Submission<T, U> = {
 const submissions = useSubmissions(action, (input) => filter(input));
 const submission = useSubmission(action, (input) => filter(input));
 ```
+##  Revalidate cached functions
+
+###  Revalidate all (default)
+By default all cached function will be revalidated wether the action does not return or return a "normal" response.
+```jsx
+
+const deleteTodo = action(async (formData: FormData) => {
+  const id = Number(formData.get("id"))
+  await api.deleteTodo(id)
+  // return new  Response("success",  {  status:  200  });
+  // ...
```

Comment:
It's preffered that snippets are copy-pasteable. And in this case I think the return would be ideal to have. What do you think?

```suggestion
  // ...
  return new  Response("success",  {  status:  200  });
```